<launch>
  <node name="radbot_controller" pkg="radbot_controller" type="radbot_controller" output="screen">
        <param name="serial_port_name" type="str" value="/dev/ttyUSB0"/>
        <param name="loam_Frame_id" type="str" value="rover_body"/>
        <param name="controller_update_freq" type="double" value="40.0"/>

        <param name="steering_angle_threshold" type="double" value="0.2"/>
        <param name="steering_kp" type="double" value="45.0"/>
        <param name="steering_ki" type="double" value="45.0"/>
        <param name="steering_kd" type="double" value="0.05"/>
        <param name="steering_out_limit" type="double" value="50.0"/>
        <param name="steering_not_moving_gain" type="double" value="1.0"/>
        <param name="steering_not_moving_offset" type="double" value="0.0"/>
        <param name="steering_not_moving_sum_flag" type="bool" value="true"/>
        <param name="steering_not_moving_limit" type="double" value="120.0"/>
        <param name="is_turning_thresh" type="double" value="0.15"/>
        <param name="position_kp" type="double" value="35.0"/>
        <param name="position_ki" type="double" value="8.0"/>
        <param name="position_kd" type="double" value="4.0"/>
        <param name="position_out_limit" type="double" value="20.0"/>
        <param name="position_not_moving_gain" type="double" value="1.0"/>
        <param name="position_not_moving_offset" type="double" value="0.0"/>
        <param name="position_not_moving_sum_flag" type="bool" value="true"/>
        <param name="position_not_moving_limit" type="double" value="120.0"/>
        <param name="is_moving_thresh" type="double" value="0.02"/>
  </node>  
</launch>